src/Microsoft.Health.Fhir.Shared.Core/Features/Operations/Everything/PatientEverythingService.cs

```diff
@@ -108,7 +110,7 @@ public class PatientEverythingService : IPatientEverythingService
             {
                 // Phase 0 gets the patient and any resources it references directly.
                 case 0:
-                    searchResult = await SearchIncludes(resourceId, parentPatientId, since, types, token, cancellationToken);
+                    searchResult = await SearchIncludes(resourceId, parentPatientId, since, types, token, !excludeLinks, cancellationToken);
```

Comment:
Good point. I took this approach because I thought it made the logic in the `SearchIncludes()` method more readable. I recognize it is a bit of a tradeoff between consistency and readability, but I think it is nice to favour readability in this file since the logic is decently complex. Thoughts?